src/commands/txCommands/status.js

```diff
@@ -0,0 +1,21 @@
+const Rx = require('rxjs');
+const txlib = require('./util');
+const chalk = require('chalk');
+
+module.exports = {
+    command: 'status',
+    description: 'get translation status of resources (branches)',
+    handler: argv => {
+        txlib.checkEnvVars();
+        console.log(chalk.blue('checking resource status\n'));
+
+        console.log('Incomplete Resources')
+        txlib.resourcesIncomplete$
+            .do(resource => { console.log(resource[0], resource[1]) })
+            .toArray() // this is a hack to make resourcesIncomplete$ stop before resourcesComplete$
```

Comment:
if you used `.toArray` instead of `.do`, it would be easier to conditionally print 'Incomplete Resources' without needing it to be a 'hack'

```js
.toArray()
.do(resources => {
  if (resources.length) {
    console.log('Incomplete Resources');
    resources.forEach(([branchName, percentage]) => console.log(branchName, percentage));
  }
})
```